data/json/recipes/practice/melee.json

```diff
@@ -63,30 +163,18 @@
     "//": "Only real and powerful cutting weapons.",
     "tools": [
       [
-        "katana",
-        "scimitar",
-        "arming_sword",
-        "broadsword",
-        "cavalry_sabre",
-        "cutlass",
-        "dao",
-        "jian",
-        "longsword",
-        "nodachi",
-        "wakizashi",
-        "zweihander",
-        "butterfly_swords",
-        "sword_xiphos",
-        "khopesh",
-        "glaive",
-        "halberd",
-        "poleaxe",
-        "battleaxe",
-        "fire_ax",
-        "ax",
-        "ji",
-        "naginata",
-        "scythe_war"
+        [ "real_shivs", 1, "LIST" ],
+        [ "real_claws", 1, "LIST" ],
+        [ "real_hooks", 1, "LIST" ],
+        [ "real_knives", 1, "LIST" ],
+        [ "basic_short_swords", 1, "LIST" ],
```

Comment:
Should this be removed or are basic short swords better than other basic categories of weapons?